soda_machine.rb

```diff
@@ -4,15 +4,21 @@ class SodaMachine
   def initialize(args = {})
     @sodas = args[:sodas]
     @cash = args[:cash]
+    
   end
 
   def current_inventory_count
+    @sodas.length
+  
   end
 
   def find_soda(soda_brand)
+    @sodas.each do |soda, price|
```

Comment:
Thanks James, maybe another session on the nested data structure would be nice.  I'll grab you or just do another mini breakout tomorrow.